handbook/engineering/distribution/index.md

```diff
@@ -7,6 +7,7 @@ The distribution team is responsible for making Sourcegraph easy to deploy, scal
 ## Contact
 
 - Slack: #distributioneers channel or @distribution
+- [Support rotation](##support-rotation)
```

Comment:
you only need one `#` right?
```suggestion
- [Support rotation](#support-rotation)
```

Reply:
It does, but I default to have the actual "level". I dont mind either way.